content/docs/reference/pkg/packet/_index.md

```diff
@@ -10,6 +10,12 @@ menu:
 <!-- WARNING: this file was generated by Pulumi Docs Generator. -->
 <!-- Do not edit by hand unless you're certain you know what you are doing! -->
 
+<div class="note note-info" role="alert">
+    <p><b>
+        Please note that the Packet provider has been deprecated. The replacement provider is the <a href="/docs/reference/pkg/equinix-metal">Equinix Metal</a> provider.
+    </b></p>
+</div>
```

Comment:
We should align the presentation to the other notes in our docs (e.g. using the icon in the callout).

```suggestion
<div class="note note-info" role="alert">
    <p>
    <i class="fas fa-info-circle pr-2"></i>
        The Packet provider has been deprecated. The replacement provider is the <a href="/docs/reference/pkg/equinix-metal">Equinix Metal</a> provider.
    </p>
</div>
```